packages/mobile/src/common/enums/ui/button-type.enum.ts

```diff
@@ -0,0 +1,7 @@
+enum ButtonType {
+  MAIN = 'main',
```

Comment:
I'd suggest splitting type and appearance into separate props/enums. So that we can make button `primary` and `outlined`, or `secondary` and `filled` etc.

Reply:
updated